lib/mutant/ast/meta/send.rb

```diff
@@ -56,6 +56,19 @@ def binary_method_operator?
           Types::BINARY_METHOD_OPERATORS.include?(selector)
         end
 
+        # Check if receiver is a top level constant
+        #
+        # @return [Boolean]
+        #
+        # @api private
+        def receiver_top_level_const?
+          return false unless receiver && receiver.type.equal?(:const)
+
+          const_parent = receiver.children.fetch(0)
+
+          const_parent.nil? || const_parent.type.equal?(:cbase)
```

Comment:
I wounder if the above two lines would better be factored out into another ast-meta? This way the `receiver.children.fetch(0)` would use the named children DSL, removing the magic constant `0`.